models/matris/join_matris_preds.py

```
@@ -33,45 +53,61 @@
-# %%
-fig = pmv.density_scatter(
-    df=df_preds, x=MbdKey.e_form_dft, y=e_form_matris_col, template="pymatviz_white"
+cse: ComputedStructureEntry
+e_col = "matris_energy"
+struct_col = "matris_structure"
+e_form_matris_col = "e_form_per_atom_matris"
+
+for mat_id in tqdm(df_matris.index):
+    matris_energy = df_matris.loc[mat_id, e_col]
+    mlip_struct = Structure.from_dict(df_matris.loc[mat_id, struct_col])
+    cse = df_wbm_cse.loc[mat_id, CSE]
```

Ruff (0.14.10)

Undefined name `CSE`

(F821)